tftpsync/susemanager-tftpsync-recv/susemanager-tftpsync-recv.spec

```diff
@@ -83,10 +83,10 @@ if [ -f /etc/apache2/conf.d/susemanager-tftpsync-recv.conf.rpmnew ]; then
             exit 1;
     fi;
 
-    SUMA_IP=$( host $PARENT_FQDN | awk '{ print $4 }' )
+    SUMA_IP=$(getent hosts $PARENT_FQDN | awk '{ print $1 }' || echo "")
 
-    sed -i "s/^[[:space:]]*Allow from[[:space:]].*$/        Allow from $SUMA_IP/" /etc/apache2/conf.d/susemanager-tftpsync-recv.conf.rpmnew
-    sed -i "s/^[[:space:]#]*Require ip[[:space:]].*$/        Require ip $SUMA_IP/" /etc/apache2/conf.d/susemanager-tftpsync-recv.conf.rpmnew
+    sed -i "s/^[[:space:]]*Allow from[[:space:]].*$/        Allow from $PARENT_FQDN/" /etc/apache2/conf.d/susemanager-tftpsync-recv.conf.rpmnew
+    sed -i "s/^[[:space:]#]*Require ip[[:space:]].*$/        Require ip $PARENT_FQDN/" /etc/apache2/conf.d/susemanager-tftpsync-recv.conf.rpmnew
```

Comment:
```suggestion
    sed -i "s/^[[:space:]#]*Require (ip|host)[[:space:]].*$/        Require host $PARENT_FQDN/" /etc/apache2/conf.d/susemanager-tftpsync-recv.conf.rpmnew
```

Reply:
I have reverted the host check. So this comment is not needed anymore.